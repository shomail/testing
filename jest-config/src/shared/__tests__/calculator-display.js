import React from 'react'
import {render} from 'react-testing-library'
import {ThemeProvider} from 'emotion-theming'
import {dark} from '../../themes'
import CalculatorDisplay from '../calculator-display'

test('mounts', () => {
  const {container} = render(
    <ThemeProvider theme={dark}>
      <CalculatorDisplay value="0" />
    </ThemeProvider>,
  )
  expect(container.firstChild).toMatchSnapshot()
})
