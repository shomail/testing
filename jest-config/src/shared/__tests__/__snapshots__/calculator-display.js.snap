// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mounts 1`] = `
<div
  class="css-1nai0zg-calculator-display--DisplayContainer emotion-0"
>
  <div
    class="autoScalingText"
    style="transform: scale(1,1);"
  >
    0
  </div>
</div>
`;
